src/main/java/duke/tasks/Event.java

```diff
@@ -0,0 +1,43 @@
+package duke.tasks;
+import java.time.*;
+import java.time.format.*;
+
+public class Event extends Task {
+    public LocalDateTime start;
+    public LocalDateTime end;
+
+    public Event (String name, LocalDateTime start, LocalDateTime end) {
+        super(name);
+        this.start = start;
+        this.end = end;
+        this.info = "E,0," + name + "," + start.getYear() + "," + start.getMonthValue() + "," + start.getDayOfMonth()
+                + "," + start.getHour() + "," + start.getMinute() + "," + end.getYear() + "," + end.getMonthValue() + "," + 
+                end.getDayOfMonth() + "," + end.getHour() + "," + end.getMinute();
+    }
+
+    @Override
+    public void mark() {
+        this.isDone = true;
+        this.info = "E,1," + name + "," + start.getYear() + "," + start.getMonthValue() + "," + start.getDayOfMonth()
+                + "," + start.getHour() + "," + start.getMinute() + "," + end.getYear() + "," + end.getMonthValue() + "," + 
+                end.getDayOfMonth() + "," + end.getHour() + "," + end.getMinute();
+    }
```

Comment:
Will it be better if these statements are abstracted into another method? 'this.info = ...' appears to have been repeated a few times!